website/src/pages/Index.vue

```diff
@@ -27,7 +27,7 @@
                 laptop inside Docker.
               </p>
               <div class="flex-1 text-center pb-4 m-0">
-                <a href="/docs/v0.8/introduction/quickstart">
+                <a href="/docs/latest/">
```

Comment:
Can we make `/docs/latest/introduction/quickstart` work?

Reply:
We _can_, but we need to commit to a URL that we think makes sense long term. If we rearrange the layout, will `/docs/latest/introduction/quickstart` still make sense? What about just `/docs/latest/quickstart`? Then we don't care if the QuickStart is under introduction.

Reply:
Can `/docs/latest/XXX` redirect to `/docs/v0.mm/XXX` like https://www.talos.dev/docs/latest/ redirects now?

Reply:
No, not quite. We have to define redirects statically: https://github.com/talos-systems/talos/blob/master/website/static/_redirects.

Reply:
`/docs/latest/* /docs/v0.10/:splat` worked. PTAL